openedx/features/edly/management/commands/convert_oidc_clients_to_oauth2.py

```diff
@@ -0,0 +1,80 @@
+"""
+Edly's management command to convert OIDC clients to Django OAuth Toolkit Application clients.
+"""
+
+import logging
+
+from django.core.management.base import BaseCommand
+from oauth2_provider.models import get_application_model
+from provider.oauth2.models import Client
+
+from openedx.core.djangoapps.oauth_dispatch.models import ApplicationAccess
+
+logger = logging.getLogger(__name__)
+
+Application = get_application_model()
+
+
+class Command(BaseCommand):
+    """
+    Converts OIDC clients to a Django OAuth Toolkit (DOT) Application clients.
+    """
+    help = "Converts OIDC clients to a Django OAuth Toolkit (DOT) Application clients."
+
+    def _get_oidc_clients(self):
+        """
+        Returns current OIDC clients.
+        """
+        return Client.objects.all()
+
+    def _create_or_update_application(self, user, app_name, application_kwargs):
+        """
+        Creates a new application if it does not exists otherwise update existing application.
+        """
+        application, is_created = Application.objects.update_or_create(
+            user=user, name=app_name, defaults=application_kwargs
+        )
+        logger.info('{} {} application with id: {}, client_id: {}'.format(
+            'Created' if is_created else 'Updated',
+            app_name,
+            application.id,
+            application.client_id,
+        ))
+        return application
+
+    def _create_or_update_access(self, application, application_access_kwargs):
+        """
+        Create application access scopes if it does not exists otherwise update existing access.
+        """
+        _, is_created = ApplicationAccess.objects.update_or_create(
```

Comment:
single underscore in python is used for marking strings for translation, to ignore an argument use double underscore `__`

Reply:
Done